src/main/fc/config.c

```diff
@@ -741,7 +751,7 @@ void ensureEEPROMStructureIsValid(void)
     if (isEEPROMStructureValid()) {
         return;
     }
-    resetEEPROM();
+    resetEEPROM(false);
```

Comment:
can we avoid the boolean argument?  this reads like 'do something, but don't'.

suggest `resetToCustomDefaults()` and `resetToDefaults()` this will remove all the `false` arguments from everywhere else in this PR.

Reply:
or even `resetConfigToDefaults` and `resetConfigToCustomDefaults`.  Since there's a general move to replace the word EEPROM with Config right?

Reply:
So are you suggesting to add a coding style rule 'boolean function parameters are not allowable'?

Reply:
And yeah, I thought the same about renaming EEPROM - but the scope of the rename is fairly wide, so this should be done in a separate pull request - do you want to open one?

Reply:
maybe at some point, I would like to work more on the EEPROM loading interface and cleanup, after my current PR's I have to re-focus on some paid work so won't have time for a bit.

Reply:
> 
> 
> So are you suggesting to add a coding style rule 'boolean function parameters are not allowable'?

I tried to reply to this via email but it hasn't appeared.   but the answer is no certainly not, they have a place but often it's just better to use two methods for clarity and to make the calls easier to read as it removes the 'false' and 'true' everywhere.  This was demonstrated by the recent dshot input/output code refactor which did this and eventually led to a big cleanup and the ability to make one of the two methods private to the file it was called from.

another option is to use an `enum` argument, at least then the value has a name, and not just 'true' or 'false'.  e.g. `resetEEPROM(APPLY_DEFAULTS)` and `resetEEPROM(APPLY_DEFAULTS)` which is a lot more readable in say a diff (where you have no context) than simply `resetEEPROM(true)` and `resetEEPROM(false)`.  remember there are many places where people read the code, not just in the IDE which makes it easy to look up method arguments,  code review on github for instance will only show you one or two lines and you have to go digging to find what `true` might mean by reading the source to `resetEEPROM` which is something you don't want the reader to have to do.